packages/react-composites/src/composites/CallComposite/Call.tsx

```diff
@@ -29,18 +29,38 @@ export type CallCompositeProps = {
   locale?: Locale;
   callInvitationURL?: string;
   onRenderAvatar?: (props: PlaceholderProps, defaultOnRender: (props: PlaceholderProps) => JSX.Element) => JSX.Element;
+  options?: CallOptions;
   identifiers?: Identifiers;
 };
 
+/**
+ * Additional customizations for the chat composite
+ */
+export type CallOptions = {
+  /**
+   * UNSTABLE: Feature flag for the local stream to be in a PIP (picture in picture) floating on the bottom right corner.
+   *
+   * This option will be removed once VideoGallery is stable.
```

Comment:
If we intend to remove this option when we mark the `VideoGallery` stable, I'm assuming that PIP mode will be the only choice? In that case, it'll flip default and surprise customers (closer to stable, so worse).

Should we start by defaulting to local video (and providing the option to go back to embedded local video for those customers who don't want this change suddenly)?